Autodesk/synthesis
# Refresh APS Token

Changes:
- Refresh the APS auth data when requested after it's expired
- Move code challenge generation and verifier to server

Testing:
- Sign in with APS and make sure your name and pfp show up in the main HUD
- Refresh token and see if name/pfp disappear and then reappear
- Expire token (might have to click multiple times?) and see if same thing happens as refresh token

> [!IMPORTANT]
> The server has to be running when testing, which can be found in the website repo.

> [!IMPORTANT]
> - [x] ~~Waiting on server update PR to merge~~

[JIRA Issue](https://jira.autodesk.com/browse/AARD-1699)
